apps/emqx_ds_builtin_local/src/emqx_ds_builtin_local_meta_worker.erl

```diff
@@ -51,7 +51,7 @@ start_link(DB) ->
 init([DB]) ->
     process_flag(trap_exit, true),
     S = #s{db = DB},
-    self() ! tick,
+    self() ! {?MODULE, tick},
```

Comment:
we have to many tick tick tick in the system, this just make it for easy tracing but may help to avoid ticking other systems :)